sky/setup_files/setup.py

```diff
@@ -74,6 +74,9 @@ def parse_readme(readme: str) -> str:
     # the latest version.
     'colorama<0.4.5',
     'cryptography',
+    # Cython 3.0 release breaks PyYAML and other dependencies.
+    # https://github.com/yaml/pyyaml/issues/601
+    'Cython<0.3',
```

Comment:
```suggestion
    'Cython<3',
```

Reply:
Thanks - we've changed the fix by pinning pyyaml instead.